### ISSUE TITLE
Everyplay causes a Metal error when the CAMetalLayer's pixelFormat is set to MTLPixelFormatBGRA8Unorm_sRGB

### ISSUE THREAD

**Issue by AbigailBuccaneer** [Everyplay/everyplay-ios-sdk#31]
From the [`CAMetalLayer pixelFormat` documentation](https://developer.apple.com/library/prerelease/ios/documentation/Animation/Reference/CAMetalLayer_Ref/index.html#//apple_ref/occ/instp/CAMetalLayer/pixelFormat):

> The pixel format for a Metal layer must be either `MTLPixelFormatBGRA8Unorm` (the default) or `MTLPixelFormatBGRA8Unorm_sRGB`.

In our game we use `MTLPixelFormatBGRA8Unorm_sRGB`. However, when enabling Everyplay, the following Metal API validation error is thrown as soon as a recording is started:

```
/SourceCache/AcceleratorKit/AcceleratorKit-14.10/ToolsLayers/Debug/MTLDebugBlitCommandEncoder.mm:119: failed assertion `[sourceTexture pixelFormat](MTLPixelFormatBGRA8Unorm_sRGB) must equal [destinationTexture pixelFormat](MTLPixelFormatBGRA8Unorm).'
```

**Comment by paugit** [comment]
Unfortunately only MTLPixelFormatBGRA8Unorm is currently supported with Everyplay. More supported pixel formats should become available in the near future. Thanks!

**Comment by paugit** [comment]
This is now fixed in the latest release (1.9.5). Now any pixel format is supported however the final video will still be encoded from MTLPixelFormatRGBA8Unorm buffer so there might be some color differences between the video and screen when using _srgb profile.

**Comment by AbigailBuccaneer** [comment]
@paugit, when you say "color differences", is that that there may be color banding due to precision differences between linear and sRGB, or will the entire video be lighter/darker than the screen due to a lack of proper conversion from sRGB?
